src/modules/uxrce_dds_client/dds_topics.h.em

```diff
@@ -83,6 +83,7 @@ void SendTopicsSubs::reset() {
 	num_payload_sent = 0;
 	for (unsigned idx = 0; idx < sizeof(send_subscriptions)/sizeof(send_subscriptions[0]); ++idx) {
 		send_subscriptions[idx].data_writer = uxr_object_id(0, UXR_INVALID_ID);
+		orb_unsubscribe(fds[idx].fd);
```

Comment:
```suggestion
		orb_unsubscribe(fds[idx].fd);
		fds[idx].fd = -1;
```

This might not be necessary depending on how it's used after reset, but doesn't hurt.

Reply:
Yes, good idea. I added the suggestion.